Sources/SpeziOnboarding/OnboardingStack.swift

```diff
@@ -0,0 +1,18 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+// swiftlint:disable file_types_order
+
+import SpeziViews
+
+/// Managed, sequential presentation of consecutive ``OnboardingView``s
+public typealias OnboardingStack = ManagedNavigationStack
```

Comment:
I would even argue to put a deprecation warning on these typealiases (still, makes sense to keep them for a sort of backwards compatibility, we're still breaking API as far as I see) and use them as little as possible within the entire SpeziOnboarding package. The new `ManagedNavigationStack` is not a part of onboarding anymore, we shouldn't treat it as such or give anyone the idea that it is.